.gitignore

```diff
@@ -1,2 +1,22 @@
 random_stuff/*
-__pycache__/**
\ No newline at end of file
+__pycache__/**
+
+# Folder view configuration files
+.DS_Store
+Desktop.ini
+
+# Thumbnail cache files
+._*
+Thumbs.db
+
+# Files that might appear on external disks
+.Spotlight-V100
+.Trashes
+
+# Compiled Python files
+*.pyc
+
+venv
+
+.ipynb_checkpoints
+*/.ipynb_checkpoints/*
```

Comment:
Isn't the first line enough?
`If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.` https://git-scm.com/docs/gitignore